documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -16,7 +16,7 @@ We do think we have the basics covered and we are looking for feedback.
 
 Patches can be sent as pull requests to the https://github.com/hibernate/hibernate-search[Github repository],
 but also general feedback, suggestions and questions are very welcome.
-To get in touch or find other interesting links for contributors, see the http://hibernate.org/community/[Hibernate Community].
+To get in touch or find other interesting links for contributors, see the http://hibernate.org/community/[Hibernate Community] page.
```

Comment:
In the context of a user reading this on the PDF, calling it "page" is a bit confusing no?